R/RLearner_fcregr_Arima.R

```diff
@@ -0,0 +1,90 @@
+#'@export
+makeRLearner.fcregr.Arima = function() {
+  makeRLearnerForecastRegr(
+    cl = "fcregr.Arima",
+    package = "forecast",
+    par.set = makeParamSet(
+      makeIntegerVectorLearnerParam(id = "order", len = 3L,
+                                    lower = 0L, upper = Inf,
+                                    default = c(0L,0L,0L)),
+      makeIntegerVectorLearnerParam(id = "seasonal", len = 3L,
+                                    lower = 0L, upper = Inf,
+                                    default = c(0L,0L,0L)),
+      makeLogicalLearnerParam(id = "include.mean", default = TRUE),
+      makeLogicalLearnerParam(id = "include.drift", default = FALSE),
+      makeNumericLearnerParam(id = "lambda", default = NULL, special.vals = list(NULL), when = "both"),
+      makeLogicalLearnerParam(id = "biasadj", default = FALSE, when = "both"),
+      makeDiscreteLearnerParam(id = "method", values = c("CSS-ML", "ML", "CSS"), default = "CSS-ML"),
+      makeUntypedLearnerParam(id = "model", default = NULL),
+      # arima params
+      makeLogicalLearnerParam(id = "transform.pars", default = TRUE),
+      makeNumericVectorLearnerParam(id = "fixed", len = NA, default = NULL, special.vals = list(NULL)),
+      makeNumericVectorLearnerParam(id = "init", len = NA, default = NULL, special.vals = list(NULL)),
+      # No default
+      makeIntegerLearnerParam("n.cond", lower = 0),
+      makeDiscreteLearnerParam("SSinit", values = c("Gardner1980", "Rossignol2011"), default = "Gardner1980", tunable = FALSE),
+      makeDiscreteLearnerParam("optim.method", default = "BFGS", values = c("Nelder-Mead", "BFGS",
+                                                                            "CG", "L-BFGS-B",
```

Comment:
indentation is wrong, just configure your editor properly and use something like "Re-indent Lines" like in RStudio.